### PR TITLE
`number-max-precision` now better handle case insensitive `@import` at-rules

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @davidtheclark @jeddy3

### CODE REVIEW

**Comment by ntwb** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - Fixed: `selector-max-specificity` no longer crashes on selectors containing `:not()` pseudo-classes.
 - Fixed: `time-no-imperceptible` correctly handles negative time.
 - Fixed: `function-calc-no-unspaced-operator` correctly interprets negative fractional numbers without leading zeros and those wrapped in parentheses.
+- Fixed: `number-max-precision` now better handle case insensitive `@import` at-rules.
```
Comment (jeddy3) [inline CHANGELOG.md]: Can we make this explicit e.g. 

- Fixed: `number-max-precision` now ignores lowercase and mixed case `@import` at-rules.

Comment (evilebottnawi) [inline CHANGELOG.md]: @jeddy3 before we ignore only lowercase, right `now ignore uppercase and mixed case`

**Comment by jeddy3** [comment]
LGTM.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 - Fixed: `selector-max-specificity` no longer crashes on selectors containing `:not()` pseudo-classes.
 - Fixed: `time-no-imperceptible` correctly handles negative time.
 - Fixed: `function-calc-no-unspaced-operator` correctly interprets negative fractional numbers without leading zeros and those wrapped in parentheses.
+- Fixed: `number-max-precision` now ignores uppercase and mixed case @import at-rules.
```
Comment (davidtheclark) [inline CHANGELOG.md]: Please add ticks around `@import`.

**Comment by davidtheclark** [comment]
👍

**Pull request merged**